zee/src/components/theme/base16.rs

```diff
@@ -578,3 +642,28 @@ pub const ZENBURN: Base16Theme = Base16Theme {
     base0e: Colour::rgb(220, 140, 195),
     base0f: Colour::rgb(147, 224, 227),
 };
+
+pub const VSCODE_DARK: Base16Theme = Base16Theme {
+    base00: Colour::rgb(0x1E, 0x1E, 0x1E),
+    base01: Colour::rgb(0x3A, 0x3D, 0x41),
+    base02: Colour::rgb(0x26, 0x4F, 0x78),
+    base03: Colour::rgb(0x6A, 0x99, 0x55),
+    base04: Colour::rgb(0x00, 0x7A, 0xCC),
+    base05: Colour::rgb(0x9C, 0xDC, 0xFE),
+    base06: Colour::rgb(0xFF, 0xFF, 0xFF),
+
+    // TODO:  Define correct colour!
+    base07: Colour::rgb(0x00, 0x00, 0x00),
+
+    base08: Colour::rgb(0x56, 0x9C, 0xD6),
+    base09: Colour::rgb(0xD2, 0x1A, 0x25),
+    base0a: Colour::rgb(0x4E, 0xC9, 0xB0),
+    base0b: Colour::rgb(0xB5, 0xCE, 0xA8),
+    base0c: Colour::rgb(0xCE, 0x91, 0x78),
+    base0d: Colour::rgb(0xDC, 0xDC, 0xAA),
+
+    // TODO:  Define correct colour!
+    base0e: Colour::rgb(0x00, 0x00, 0x00),
```

Comment:
Yes, I was unable to deduce these colours.

I started documenting the colours since I noticed that the colour intended for variables is actually applied on keywords.  So I tried to set all colours according to VS Code's Dark Mode but these two colours were not applied at all.  At least I was unable to find them.  Setting them to red (`0xFF0000`) did not change anything regarding the colour theme so I left them black and marked them `TODO`.

Reply:
:+1: I see, cool, may be worthing adding "currently unused" to the comment 

> the colour intended for variables is actually applied on keywords

Ooops, I definitely took a bit of liberty with using the colours -- part of refactoring theme management (in addition to make it configurable not in source code), we should revisit how the colours are used for base16 themes.

Reply:
Okay, I will adjust the documentation accordingly. :wrench:

Reply:
Done. :wrench: :heavy_check_mark: